messaging/connectors/kafka/src/main/java/module-info.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module io.helidon.microprofile.connectors.kafka {
+    requires java.logging;
+
+    requires static cdi.api;
+    requires static javax.inject;
+    requires static java.activation;
+    requires static kafka.clients;
+    requires io.helidon.microprofile.config;
```

Comment:
Helidon MP implementations must not be required in an SE module.
If you want to split into SE/MP, then SE modules can only depend on other SE modules.
SE modules can depend on MP APIs and SPIs of the specifications.